distuv/norm_example_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv_test
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/gonum/stat"
+	"github.com/gonum/stat/distuv"
+)
+
+func ExampleNormal() {
+	// Create a normal distribution
+	dist := distuv.Normal{
+		Mu:    2,
+		Sigma: 5,
+	}
+
+	data := make([]float64, 100000)
+
+	// Draw some random values from the standard normal distribution
+	for i := range data {
+		data[i] = dist.Rand()
+	}
+
+	mean, std := stat.MeanStdDev(data, nil)
+	meanErr := stat.StdErr(std, float64(len(data)))
+	stdErr := std * math.Sqrt(2/(float64(len(data))-1))
```

Comment:
Does this actually help the example? It seems to me that the example should just be showing how to draw a sample, so it would be enough to

```
	mean, std := stat.MeanStdDev(data, nil)
	meanErr := stat.StdErr(std, float64(len(data)))
	
	fmt.Printf("mean=    %1.1f ± %0.1v\n", mean, meanErr)
```